vignettes/advanced.Rmd

```diff
@@ -268,15 +466,161 @@ pool(anl_obj)
 
 # Delta Adjustment
 
+A delta-adjustment refers to a shift in the distribution of the (unobserved) outcome values by a certain amount. This might be used to perform sensitivity analyses.
+In the context of multiple imputation (MI), this is typically done by adding a fixed amount to the imputed values used in the primary analysis.
+This implies that the delta-adjustment can be performed prior to the third stage of the MI process: The fit of the analysis model (`analyse()`).
+That is, there is no additional consistent computational time needed to perform this type of sensitivity analysis since there is no need to re-fit the imputation model.
```

Comment:
Not sure "consistent" is the right word here. Perhaps just re-phrase as, "this means this type of sensitivity analysis can be perforemed without having to re-fit the imputation model"